schema/cosa/schema_doc.go

```diff
@@ -23,6 +23,13 @@ var generatedSchemaJSON = `{
              "type":"number",
              "title":"Size in bytes"
             },
+           "skip-compression": {
+             "$id": "#/artifact/skip-compression",
+             "type":"boolean",
+             "title":"Skip compression",
+             "description":"Artifact should not be automatically compressed or decompressed",
```

Comment:
totally optional: I'm trying to think of something that has more meaning here but am failing to do so and keep it succinct.

```suggestion
             "description":"Artifact should not be compressed or decompressed before use",
```